### PR TITLE
More clang-tidy fixes

### PR DESCRIPTION
**Pull request by itopcuoglu**

### CODE REVIEW

File: src/TiogaMeshInfo.h [hunk 1]
```diff
@@ -24,7 +24,7 @@ struct MeshBlockInfo
     static constexpr int max_vertex_types = 4;
     using GlobalIDType = uint64_t;
 
-    enum QVarType { ROW = 0, COL };
+    enum QVarType : uint8_t { ROW = 0, COL };
```
Comment (jrood-nrel) [inline src/TiogaMeshInfo.h]: I think `COL` probably should have a value here.

File: src/codetypes.h [hunk 2]
```diff
@@ -55,22 +54,22 @@ typedef int32_t qcoord_t;
...
-#define INTERSECT_ALG                                                          \
-    1 // [0] point-box inclusion only
-      // [1] face-box intersection (water-tight)
-
-#define NON_UNIQUE_NODES                                                       \
-    1 // [0] wbc nodes are NOT listed as obc nodes
-      // [1] wbc nodes may also be listed as obc nodes
+//    1 // [0] point-box inclusion only
+//      // [1] face-box intersection (water-tight)
+enum : uint8_t { INTERSECT_ALG = 1 };
```
Comment (jrood-nrel) [inline src/codetypes.h:68]: I'm not really sure if enums can translate directly to preprocessor defines.

File: src/codetypes.h [hunk 3]
```diff
@@ -55,22 +54,22 @@ typedef int32_t qcoord_t;
...
 
+//    1 // [0] wbc nodes are NOT listed as obc nodes
+//      // [1] wbc nodes may also be listed as obc nodes
+enum : uint8_t { NON_UNIQUE_NODES };
```
Comment (jrood-nrel) [inline src/codetypes.h]: I think this needs a value of 1.

File: src/TiogaMeshInfo.h (hunk 1 above)
Comment (itopcuoglu) [reply src/TiogaMeshInfo.h]: I set `COL` to 1.

File: src/codetypes.h (hunk 3 above)
Comment (itopcuoglu) [reply src/codetypes.h]: It appears that I overlooked this. It's fixed now.

**Review by jrood-nrel — APPROVED** [review]

**Pull request merged**